oxidation/bindings/web/src/utils.rs

```diff
@@ -1,12 +0,0 @@
-// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
-
-pub fn set_panic_hook() {
```

Comment:
There is no need for this at all ? or are we no yet using it but might in the future ? (in any way if this is not currently needed I'm okay to remove this)

Reply:
the rust code in `binding_web` currently is the default generated by `wasm-pack` so we could easily retrieve what we've lost